woocommerce-services.php

```diff
@@ -543,8 +543,12 @@ public function init_core_wizard_payments_config() {
 				$response = $this->stripe->create_account( $email, $country );
 
 				if ( is_wp_error( $response ) ) {
-					// TODO handle case of existing account
-					$this->logger->debug( $response, __CLASS__ );
+					// Handle existing account case.
+					if ( false !== strpos( $response->get_error_message(), 'Account already exists for the provided email.' ) ) {
+						WC_Connect_Options::update_option( 'banner_stripe', 'yes' );
+					} else {
+						$this->logger->log( $response, __CLASS__ );
```

Comment:
What do you think about logging even when the message triggers the banner?

Reply:
Done in 6b4bed8

Reply:
Ah, I originally decided not to, as the way the current flow is, it's not really an error from the merchant's perspective. But of course, this isn't strictly an error log. Changed in 922dc1d